### PR TITLE
New main menu

### PR DESCRIPTION
**Pull request by deepakpaul77**
created Main menu with functionality for start and quit button

### CODE REVIEW

File: Source/Redemption/Public/MainMenuWidget.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	UButton* QuitButton;
+	
+	UFUNCTION(BlueprintCallable)
+	void Start();
+
+	UFUNCTION(BlueprintCallable)
+	void Quit();
+
+public:
+	virtual void NativeConstruct() override;
```
Comment (Entwistle-905) [inline Source/Redemption/Public/MainMenuWidget.h:31]: on our code standard we need to put public functions before protected functions

File: Source/Redemption/Private/MainMenuWidget.cpp [hunk 2]
```diff
@@ -0,0 +1,9139 @@
...
+
+	StartButton->OnClicked.AddDynamic(this, &UMainMenuWidget::Start);
+	QuitButton->OnClicked.AddDynamic(this, &UMainMenuWidget::Quit);
+	APlayerController* Controller =  UGameplayStatics::GetPlayerController(GetWorld(), 0);
+	Controller->SetInputMode(FInputModeUIOnly());
+	Controller->SetShowMouseCursor(true);
+	Controller->SetInputMode(FInputModeGameOnly());
+
+}
+
```
Comment (Entwistle-905) [inline Source/Redemption/Private/MainMenuWidget.cpp:36]: there are a lot of empty lines please delete them.

**Review by Entwistle-905 — CHANGES_REQUESTED** [review]

**Comment by thakkarnandish** [comment]
image
 I got an error when I tried to run the menu level, I might be doing it wrong??

**Comment by mac2648** [comment]
I tried it on my end and it worked fine

File: Content/UI/MenuGameMode.uasset
[the remark this answers is not in the document]
Comment (Entwistle-905) [reply Content/UI/MenuGameMode.uasset:1]: Recourses😂

[the remark this answers is not in the document]
Comment (mac2648) [reply Content/UI/MenuGameMode.uasset:1]: someone created the folder with a typo, for the first time it was not me

**Review by mac2648 — APPROVED** [review]

**Review by Entwistle-905 — APPROVED** [review]

**Pull request merged**